source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/MessageHub/Models/AvailableOperationReceiptData/AvailableChargePriceOperationConfirmationsFactoryTests.cs

```diff
@@ -56,7 +56,7 @@ public class AvailableChargePriceOperationConfirmationsFactoryTests
                 marketParticipantRepository, meteringPointAdministrator, document.Sender, actorId);
 
             var chargePriceOperationsConfirmedEvent =
```

Comment:
```suggestion
            var chargePriceOperationsAcceptedEvent =
```
and maybe factory (sut) and this test class should be renamed, too.

Reply:
done